pkg/piecestore/psclient/client.go

```diff
@@ -31,17 +30,9 @@ var ClientError = errs.Class("piecestore client error")
 var (
 	defaultBandwidthMsgSize = 32 * memory.KB
 	maxBandwidthMsgSize     = 64 * memory.KB
+	// TODO: make these configurable flags, removed global flag for now
```

Comment:
what's the reason to avoid these global flags? for these two values, we definitely don't know enough to know if we've landed on the right choices here yet. what harm does the init function cause?

Reply:
Anyways, removed changes in piecestore and will make a proper PR to fix them.